### ISSUE TITLE
If i want to render NOCS map in blender 3.1,Is it ok?

### ISSUE THREAD

**Issue by yuheyuan** [hughw19/NOCS_CVPR2019#62]
I use Blenderproc to generate datasets, however, it version is blender 3.1
however,

`bpy.context.scene.render.alpha_mode = 'TRANSPARENT' # in ['TRANSPARENT', 'SKY']`
the code in your nocs_map_cube.py, this code is not support higher version.
So, i want to aks you if you have some problem or 
some way can change 
`bpy.context.scene.render.alpha_mode = 'TRANSPARENT' # in ['TRANSPARENT', 'SKY']`

**Comment by ran894694447** [comment]
@yuheyuan  Hi, do you have a way to generate a coord image?

**Comment by yuheyuan** [comment]
> @yuheyuan Hi, do you have a way to generate a coord image?

yes, You can use https://github.com/DLR-RM/BlenderProc, use BlenderProc, this can render NOCS map

**Comment by ran894694447** [comment]
Thank you,I will try it!

**Comment by JAISON96** [comment]
@yuheyuan  Hi, I have tried to generate NOCS map for my own dataset using BlenderProc. I tried with the script provided for generating it, but it always throws this error:
line 32: vcol_layer.data[loop_index].color = color
ValueError: bpy_struct: item.attr = val: sequences of dimension 0 should contain 4 items, not 3.

Could you please hep me out to solve this. I am beginner and just recently started using blender.
